doc/man/dune-build.1

```diff
@@ -0,0 +1,346 @@
+           again at the end of the build, in a deterministic order.
+
+       -f, --force
+           Force actions associated to aliases to be re-executed even if
+           their dependencies haven't changed.
+
+       --file-watcher=VAL (absent=automatic)
+           Mechanism to detect changes in the source. Automatic to make dune
+           run an external program to detect changes. Manual to notify dune
+           that files have changed manually."
+
+       --passive-watch-mode
+           Similar to [--watch], but only start a build when instructed
+           externally by an RPC.
+
+       --react-to-insignificant-changes
+           react to insignificant file system changes; this is only useful
+           for benchmarking dune
+
+       --sandbox=VAL (absent DUNE_SANDBOX env)
+           Sandboxing mode to use by default. Some actions require a certain
+           sandboxing mode, so they will ignore this setting. The allowed
+           values are: none, symlink, copy, hardlink.
+
+       -w, --watch
+           Instead of terminating build after completion, wait continuously
+           for file changes.
+
+       --wait-for-filesystem-clock
+           Dune digest file contents for better incrementally. These digests
+           are themselves cached. In some cases, Dune needs to drop some
+           digest cache entries in order for things to be reliable. This
+           option makes Dune wait for the file system clock to advance so
+           that it doesn't need to drop anything. You should probably not
+           care about this option; it is mostly useful for Dune developers to
+           make Dune tests of the digest cache more reproducible.
+
+COMMON OPTIONS
+       These options are common to all commands.
+
+       --always-show-command-line
+           Always show the full command lines of programs executed by dune
+
+       --auto-promote
+           Automatically promote files. This is similar to running dune
+           promote after the build.
+
+       --build-dir=FILE (absent DUNE_BUILD_DIR env)
+           Specified build directory. _build if unspecified
+
+       --cache=VAL (absent DUNE_CACHE env)
+           Enable or disable Dune cache (either enabled or disabled). Default
+           is `disabled'.
+
+       --cache-check-probability=VAL (absent DUNE_CACHE_CHECK_PROBABILITY
+       env)
+           Check build reproducibility by re-executing randomly chosen rules
+           and comparing their results with those stored in Dune cache. Note:
+           by increasing the probability of such checks you slow down the
+           build. The default probability is zero, i.e. no rules are checked.
+
+       --cache-storage-mode=VAL (absent DUNE_CACHE_STORAGE_MODE env)
+           Dune cache storage mode (one of auto, hardlink or copy). Default
+           is `auto'.
+
+       --config-file=FILE
+           Load this configuration file instead of the default one.
```

Comment:
```suggestion
           Loads this configuration file instead of the default one.
```
Perhaps all of these definitions need the verb to take the singular form, since you’re talking about a singular command. Again, perhaps it would be best to put the command in monospace, so it reads:

`--config-file=FILE` loads this configuration file instead of the default one. 

(If this is what you mean to say. If not, perhaps further clarification is needed.)

Reply:
I won’t suggest any verb changes on the rest of these until the above is clarified in formatting and meaning, so you won’t have to reject them all if I misunderstood! 

Let me know, and I’ll go through them again.